File: maria/lua/create/maria_create.lua
#! /usr/bin/lua
-- ----------------------------------------------------
--	maria_create.lua
--
--					Nov/17/2014
-- ----------------------------------------------------
require	("luasql.mysql")
require	("text_manipulate")
require	("sql_manipulate")
-- ----------------------------------------------------
print ("*** 開始 ***")

local dbname = "city"
local dbhost = "localhost"
local dbuser = "scott"
local dbpass = "tiger"

local env = assert(luasql.mysql())
local con = assert(env:connect(dbname, dbuser, dbpass, dbhost))

sql_drop_table_proc	(con)
sql_create_table_proc	(con)

sql_insert_proc	(con,"t3321","岡山",578432,"1981-9-15")
sql_insert_proc	(con,"t3322","倉敷",429637,"1981-8-26")
sql_insert_proc	(con,"t3323","津山",984531,"1981-7-28")
sql_insert_proc	(con,"t3324","玉野",576439,"1981-3-29")
sql_insert_proc	(con,"t3325","笠岡",745638,"1981-1-14")
sql_insert_proc	(con,"t3326","井原",352186,"1981-5-21")
sql_insert_proc	(con,"t3327","総社",813974,"1981-10-5")
sql_insert_proc	(con,"t3328","高梁",238416,"1981-8-9")
sql_insert_proc	(con,"t3329","新見",721534,"1981-4-12")

con:close()
env:close()

print ("*** 終了 ***")
-- ----------------------------------------------------

File: maria/php/create/maria_create.php
#! /usr/bin/php
<?php
//
//	maria_create.php
//
//					Jun/29/2017
//
// ----------------------------------------------------------------
$path="../../../common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include "text_manipulate.php";
include "sql_manipulate.php";
// ----------------------------------------------------------------
function data_prepare_proc ()
{
$dict_aa = array ();

$dict_aa = dict_append_proc ($dict_aa,'t3321','岡山',725139,'2002-2-9');
$dict_aa = dict_append_proc ($dict_aa,'t3322','倉敷',417328,'2002-5-15');
$dict_aa = dict_append_proc ($dict_aa,'t3323','津山',891654,'2002-7-21');
$dict_aa = dict_append_proc ($dict_aa,'t3324','玉野',265981,'2002-11-12');
$dict_aa = dict_append_proc ($dict_aa,'t3325','笠岡',284597,'2002-4-24');
$dict_aa = dict_append_proc ($dict_aa,'t3326','井原',671942,'2002-2-8');
$dict_aa = dict_append_proc ($dict_aa,'t3327','総社',265481,'2002-8-3');
$dict_aa = dict_append_proc ($dict_aa,'t3328','高梁',792356,'2002-3-14');
$dict_aa = dict_append_proc ($dict_aa,'t3329','新見',415892,'2002-1-15');

	return	$dict_aa;
}

// ----------------------------------------------------------------
fputs (STDERR,"*** 開始 ***\n");

$dict_aa = data_prepare_proc ();

$host = 'localhost';
$user = 'scott';
$password = 'tiger123';
$db = 'city';
$dsn = 'mysql:dbname=' . $db . ';host=' . $host . ';charset=utf8';

try
	{
	$dbcon = new PDO ($dsn, $user,$password);
	drop_table_proc ($dbcon);
	create_table_proc ($dbcon);
	dict_to_db_proc ($dict_aa,$dbcon);
	$dbcon = null;
	}
catch  (PDOException $e)
	{
	print('Error:'.$e->getMessage());
	}

fputs (STDERR,"*** 終了 ***\n");
// ----------------------------------------------------------------
?>
